docker/client/Dockerfile

```diff
@@ -21,5 +18,3 @@ COPY ./client /app
 # CMD commands get executed at container runtime!
 RUN chmod +x /entrypoint.sh
 CMD ["/entrypoint.sh"]
-
-# TODO: Production 
```

Comment:
Typically a prod build would have nextjs build as "standalone" and that would be then copied inside the container, but this should be fine for most purposes